configs/sst_desktop_platform_technologies-gnome-desktop-eln.yaml

```diff
@@ -0,0 +1,31 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: GNOME Desktop
+  description: GNOME Desktop core packages
```

Comment:
Is having two workloads in ELN with the same name/description going to cause any problems?

Reply:
https://github.com/minimization/content-resolver/blob/master/config_specs/workload.yaml#L20 doesn't mention that it's supposed to be unique.

Reply:
We're going to merge it and find out one way or another.  Can fix it up if needed.